data/data/powervs/loadbalancer/alb.tf

```diff
@@ -0,0 +1,151 @@
+################################################################
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Licensed Materials - Property of IBM
+#
+# ©Copyright IBM Corp. 2021
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################
+
+locals {
+  api_servers       = concat([var.bootstrap_ip], var.master_ips)
+  api_servers_count = 4 # bootstrap + master 
+  app_servers       = var.master_ips
+  app_servers_count = 3
+}
+
```

Comment:
And also we need to have one private load balancer for the internal communication(from workers => masters) for 6443 port

Reply:
@mkumatag is it ok if I address this is different PR? Need to test few scenarios based on the hostname generated and accessibility.

Reply:
make sense, will create a separate PR for that.